platform-sdk/swirlds-common/src/main/java/com/swirlds/common/io/config/FileSystemManagerConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2016-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.io.config;
+
+import com.swirlds.common.io.filesystem.FileSystemManager;
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.ConfigProperty;
+import java.time.Duration;
+
+/**
+ * Settings for {@link FileSystemManager}
+ *
+ * @param rootPath
```

Comment:
Unnecessary newline makes this a little odd.

Reply:
if I need to add a change in this PR, I will fix this; if not, I'll use a future PR to address it.
Thanks for spotting.